src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -609,7 +573,15 @@ def launch_fluent(
 
     new_session, meshing_mode, argvals, mode = _get_session_info(argvals, mode)
     _raise_exception_g_gu_in_windows_os(additional_arguments)
-    if _start_instance(start_instance):
+
+    if pypim.is_configured():
+        flunet_launch_mode = LaunchMode.LAUNCH_FLUENT_PIM
```

Comment:
```suggestion
        fluent_launch_mode = LaunchMode.LAUNCH_FLUENT_PIM
```
Small typo, needs to be replaced in multiple places